src/components/LooMap/LooMap.tsx

```diff
@@ -279,7 +279,7 @@ const LooMap: React.FC<LooMapProps> = ({
         `}
       >
         <TileLayer
-          attribution="&copy; <a href='https://www.openstreetmap.org/copyright'>OpenStreetMap</a> contributors"
+          attribution="&copy; <a href='https://www.openstreetmap.org/copyright'>OpenStreetMap</a> contributors | <a href='https://vercel.com/?utm_source=public-convenience-ltd&utm_campaign=oss'>Powered by Vercel &#9650;</a>"
```

Comment:
This is weird though. This attribution refers to the map tiles. Do you think they really meant this?

Reply:
as discussed, removed this and now reinstated a small mobile footer